qualtran/bloqs/mean_estimation/mean_estimation_operator_test.py

```diff
@@ -282,8 +283,9 @@ def test_mean_estimation_operator_consistent_protocols():
         _ = mean_gate.controlled(num_controls=2)
 
     # Test with_power
-    assert mean_gate.with_power(5) ** 2 == MeanEstimationOperator(
-        code, arctan_bitsize=arctan_bitsize, power=10
+    assert (
+        Power(mean_gate, 5) ** 2
```

Comment:
Does this work?

```suggestion
        (mean_gate ** 5) ** 2
```

Reply:
Yep this also works. I actually added these tests for `Power` in `util_bloqs_test.py`, so maybe we can remove this one here?